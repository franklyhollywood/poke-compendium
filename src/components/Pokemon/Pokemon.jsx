import React from 'react';

export default function Pokemon({
	pokemon,
	hp,
	ability_1,
	egg_group_1,
	pokebase,
	url_image,
}) {
	return (
		<div aria-label='pokemon' key={pokemon}>
			<p>Name: {pokemon}</p>
			<p>Health Points: {hp}</p>
			<p>Ability: {ability_1}</p>
			<p>Egg Group: {egg_group_1}</p>
			<p>Pokebase: {pokebase}</p>
			<img src={url_image} alt={pokemon} />
		</div>
	);
}
