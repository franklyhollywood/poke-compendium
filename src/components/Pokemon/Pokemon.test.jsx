import { screen, render } from '@testing-library/react';
import Pokemon from './Pokemon.jsx';

it('sort', () => {
	render(
		<Pokemon
			pokemon='Pikachu'
			hp='23'
			ability_1='compound-eyes'
			egg_group_1='bug'
			pokebase='butterfree'
		/>
	);
	const pokemon = screen.getByLabelText('pokemon');
	expect(pokemon).toMatchSnapshot();
});
