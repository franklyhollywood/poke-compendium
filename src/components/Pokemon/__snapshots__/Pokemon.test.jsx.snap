// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`sort 1`] = `
<div
  aria-label="pokemon"
>
  <p>
    Name: 
    Pikachu
  </p>
  <p>
    Health Points: 
    23
  </p>
  <p>
    Ability: 
    compound-eyes
  </p>
  <p>
    Egg Group: 
    bug
  </p>
  <p>
    Pokebase: 
    butterfree
  </p>
  <img
    alt="Pikachu"
  />
</div>
`;
